src/loaders/zarr/pixel-source.ts

```diff
@@ -38,14 +38,19 @@ interface ZarrTileSelection {
 class ZarrPixelSource<S extends string[]> implements PixelSource<S> {
   private _data: ZarrArray;
   private _indexer: ZarrIndexer<S>;
+  private _readChunks: boolean;
 
-  constructor(data: ZarrArray, public labels: Labels<S>) {
+  constructor(
+    data: ZarrArray,
+    public labels: Labels<S>,
+    public tileSize: number
```

Comment:
Is it worth having some kind of default here, at least for API stability (not that we have so many users)?  This would be a breaking change.

Reply:
I'd say no. There is no default for the tiff either. It's technically a breaking change but 1.) ZarrPixelSource is totally new as of 0.9.0, 2.) we didn't even export ZarrPixelSource correctly until 0.9.1, and 3.) I think vizarr is the only "user" of this class that I'm aware of.